gtsfm/runner/run_scene_optimizer.py

```diff
@@ -56,17 +59,30 @@ def run_scene_optimizer(args: argparse.Namespace) -> None:
         help="maximum number of consecutive frames to consider for matching/co-visibility",
     )
     parser.add_argument(
-        "--num_workers", type=int, default=1, help="Number of workers to start (processes, by default)",
+        "--num_workers",
+        type=int,
+        default=1,
+        help="Number of workers to start (processes, by default)",
```

Comment:
can be reverted as formatting change. This thing always flip-flops between one style and other in reviews :P

Reply:
I wonder why this happens with black. I'm not too worried about it, though.